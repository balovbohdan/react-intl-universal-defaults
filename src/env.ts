import { Env } from './types';

const env: Env = {
  defaultLocale: {},
  defaultLanguageTag: 'en',
  initialized: false,
};

export default env;
