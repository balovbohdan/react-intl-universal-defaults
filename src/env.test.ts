import env from './env';

test('env', () => {
  expect(env).toMatchSnapshot();
});
