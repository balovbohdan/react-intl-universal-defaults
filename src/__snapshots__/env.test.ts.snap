// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`env 1`] = `
Object {
  "defaultLanguageTag": "en",
  "defaultLocale": Object {},
  "initialized": false,
}
`;
